payments/verifier_test.go

```diff
@@ -0,0 +1,19 @@
+package payments
+
+import (
+	"context"
+	"errors"
+	"testing"
+)
+
+func TestLookupVerifier(t *testing.T) {
```

Comment:
Can we add a happy path test here with one of the two publicKeys that are valid?

Reply:
yes @husobee to add test for two valid keys